### ISSUE TITLE
About meshdatav3_feat.

### ISSUE THREAD

**Issue by huamo555** [PKU-RL/ResDex#11]
Hello author, how did you generate the 64-dimensional features in meshdatav3_feat?

**Comment by analyst-huang** [comment]
Hi @huamo555 , the 64-dimensional features are generated by UniDexGrasp. The authors used a pretrained point cloud encoder.

**Comment by huamo555** [comment]
> 嗨，64 维特征是由 UniDexGrasp 生成的。作者使用了预训练的点云编码器。

Thanks for your reply. Do you know the specific details of the pre-trained pointnet++? For example, did the author mention which pre-trained weights were used?

**Comment by analyst-huang** [comment]
Sorry, but I am not sure about the specific details of the pre-trained encoder. Perhaps  checking UniDexGrasp and UniDexGrasp++ 's papers and their repos can help.

**Comment by huamo555** [comment]
> Sorry, but I am not sure about the specific details of the pre-trained encoder. Perhaps checking UniDexGrasp and UniDexGrasp++ 's papers and their repos can help.

Thank you for your reply. I have another small question.

If this visual feat is not available, can the state_basd strategy trained by resdex be applied to an object that is not in the meshdatav3 test set?

**Comment by analyst-huang** [comment]
It can not be applied to objects that are not in the dataset. But you can still use the vision-based policy to test objects from other datasets, as it only use point clouds as vision input.

**Comment by huamo555** [comment]
> 它不能应用于不在数据集中的对象。但是，您仍然可以使用基于视觉的策略来测试其他数据集中的对象，因为它仅使用点云作为视觉输入。

The vision_based strategy will get a pointnet pre-trained model. Can this feature be used to generate 64-dimensional features for new objects for use in stated_based?

**Comment by analyst-huang** [comment]
No, the pointnet for the vision policy was trained during vision-based dagger process, while the pc features used in the state-based policy were generated by previous work. These two kinds of pc features have no direct relations to each other.
